packages/labs/react/README.md

```diff
@@ -55,6 +55,49 @@ React component.
 />
 ```
 
+#### Typescript
+
+Event callbacks can be typed using the included `EventHandler` type. By using a
```

Comment:
I would name the public type `EventName`. `EventHandler` sounds like it's the type of the actual event handler function.

Reply:
Done :)